VERSION

```diff
@@ -1 +1 @@
-v0.1.0-beta10
+v0.1.0+20241204.0.0
```

Comment:
Should this be just one `.0` at the end like `oxide.rs`?

```suggestion
v0.1.0+20241204.0
```

Reply:
Ah! So this is a recent change that came from dropshot https://github.com/oxidecomputer/omicron/pull/7050/files#diff-8e355f4f647bc8c52d26845412f351c2d246246825442453e135d69c4346c681R10

The version should also change in oxide.rs for this release AFAIK

Reply:
Thank you for the context!